setup/ReShade Setup.csproj

```diff
@@ -37,6 +39,12 @@
     <Prefer32Bit>false</Prefer32Bit>
   </PropertyGroup>
   <ItemGroup>
+    <Reference Include="Microsoft.WindowsAPICodePack, Version=1.1.2.0, Culture=neutral, processorArchitecture=MSIL">
```

Comment:
I'm not all too happy about adding additional dependencies and would much prefer a pure WPF implementation.